### DISCUSSION TITLE
Offline, privacy-respecting speech to text

### DISCUSSION THREAD

**Discussion by emes81** [Bnyro/RecordYou#209] [General]
### Describe your suggested feature

Futo has partially delivered on this with an excellent FOSS solution:

https://gitlab.futo.org/alex/voiceinput
https://voiceinput.futo.org/

But the Futo solution currently works within other apps only and is not integrated directly into a voice recorder app. Adding Futo's speech-to-text capabilities to RecordYou would make a voice recorded easily on par with Google's proprietary app.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Cwpute** [comment]
Well personally i think this isn't in the scope of this app, but if anyone wanted to take RecordYou and bundle it with Futo's solution as a new foss app, they would be more than welcome 👍

**Comment by Bnyro** [comment]
Converting to a discussion, but as stated out by @M00NJ that's nothing we would work on.

**Reply by shuvashish76** [reply]
Nothing to discuss here though, none of their projects are truly FOSS. They use their own custom license which isn't GNU compatible (i.e. approved by FSF/OSI).
